css/user-profile.styl

```diff
@@ -18,6 +18,16 @@
     .hero-container
       position: relative
       z-index: 1
+      margin: 0
+      max-width: initial
+
+      img.avatar
+        padding-right: 10px
```

Comment:
Apologies this wasn't caught earlier. This padding is causing the avatar to have an odd shape: 
<img width="110" alt="screen shot 2016-08-02 at 1 44 46 pm" src="https://cloud.githubusercontent.com/assets/5016731/17341001/404fce1c-58b8-11e6-8d1c-3c30f60373a5.png">